js/CornerWidgetUI.js

```diff
@@ -99,11 +101,14 @@ CornerWidgetUI.init = function (opts){
 				//configurable element to prevent loss of leads due to incorrect validation
 				CornerWidgetUI._ui_config.validation = {limit: w_conf["validation_limit"] ? w_conf["validation_limit"] : 1,
 														count: 0};
-				CornerWidgetUI._ui_config.auto_open = w_conf["auto_open"] ? w_conf["auto_open"] : 13000; //default to 13 seconds after loading to open the widget
+				CornerWidgetUI._ui_config.auto_open = w_conf["auto_open"] ? w_conf["auto_open"] : {"min": 1000, "max": 13000}; 
```

Comment:
@Atilla14 because your code later depends on `min` and `max` existing, you should explicitly check that the object loaded contains these properties